packages/trip-form/src/ModeIcon/index.js

```diff
@@ -4,11 +4,25 @@ import * as Icons from "@opentripplanner/icons";
 
 /**
  * A generic icon component that displays an icon for the specified transportation mode.
+ * If `icons` are defined, then
+ * the icon will be attempted to be used from that lookup of icons. Otherwise,
+ * an icon from the OTP-UI icons package will be returned.
  */
-const ModeIcon = props => {
-  const { mode } = props;
+const ModeIcon = ({ icons, mode }) => {
   if (!mode) return null;
-  switch (mode.toLowerCase()) {
+
+  // Check if there is a custom icon
+  if (icons && mode in icons) {
+    return icons[mode];
+  }
+
+  // From OTP-RR:lib/util/itinerary.js:
+  // Custom icon not available for the given iconId. Use the ModeIcon component
+  // to show the icon based on the iconId, but always use the default car icon
+  // for any car-based modes that didn't have custom icon.
+  const finalMode = mode && mode.startsWith("CAR") ? "CAR" : mode;
```

Comment:
This change appears to have broken the display of car modes:

<img width="766" alt="Screen Shot 2020-03-12 at 5 06 56 PM" src="https://user-images.githubusercontent.com/3112493/76577900-f1e48480-6483-11ea-9cec-bf1fae2c3604.png">

Reply:
Yes, thanks for catching that.